test/src/test/java/lib/form/RepeatableTest.java

```diff
@@ -634,7 +634,7 @@ private HtmlButton getHtmlButton(HtmlForm form, String buttonCaption, boolean is
      */
     private List<?> getButtonsList(HtmlForm form, String buttonCaption) {
         return form.getByXPath(
-                String.format("//button[text() = '%s'] | //button[@title = '%s']", buttonCaption, buttonCaption
+                String.format("//button[text() = '%s'] | //button[@title = '%s'] | //button[@tooltip = '%s']", buttonCaption, buttonCaption, buttonCaption
```

Comment:
Do we need an additional alternative here or could this replace the `title` one?